spec/lib/puppet_spec/compiler.rb

```diff
@@ -90,7 +90,7 @@ def eval_and_collect_notices(code, node = Puppet::Node.new('foonode'), topscope_
   # (Parameters given by name)
   #
   def evaluate(code: 'undef', source: nil, node: Puppet::Node.new('testnode'), variables: {})
-    source_location = caller[0]
+    source_location = caller(0..0).first
```

Comment:
this is so terrible..... ruby 🤦‍♀️

Reply:
Yeah, when I was reading about that Rubocop lint it was just making me sad. It makes sense in a roundabout way, but it's still _horrible_ ergonomics.